packages/falter-berlin-ssid-changer/files/hotplug.d/30-ssid-changer

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+# logger -t "ssid net/hotplug" "ssidchanger ACTION = $ACTION INTERFACE = $INTERFACE DEVICE = $DEVICE IFUPDATE_ADDRESSES = $IFUPDATE_ADDRESSES IFUPDATE_DATA = $IFUPDATE_DATA"
+
+. /lib/functions.sh
+. /lib/lib_ssid-changer.sh
+
+
```

Comment:
Can we have a uci setting which enables/disables this?  For example ffwizard.ssid_changer.enabled=1.  It can also be set up that if the setting ffwizard.ssid_changer.enabled is not found, that it defaults to enabled.  I think the ability to disable this is important.